mmdet3d/datasets/utils.py

```diff
@@ -108,3 +110,25 @@ def get_loading_pipeline(pipeline):
         'The data pipeline in your config file must include ' \
         'loading step.'
     return loading_pipeline
+
+
+def extract_result_dict(results, key):
+    """Extract and return the data corresponding to key in result dict.
+
```

Comment:
Indicate that this module essentially obtain the data from the data container and list or tuple in the description.

Reply:
Add.